### PR TITLE
Add coverage badge. Download reporter rather than including with repo

### PR DESCRIPTION
**Pull request by CCorrado**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,4 +28,4 @@ jobs:
     - stage: 'build'
       name: 'Build'
       script: "./gradlew clean lint assembleDebug jacocoTestReport -PdisablePreDex"
-      after_script: "java -jar ./codacy-coverage-reporter-4.0.5-assembly.jar report -l Kotlin -r ./guppy/build/reports/jacoco/jacocoTestReport/jacocoTestReport.xml"
\ No newline at end of file
+      after_script: "java -jar ~/codacy-coverage-reporter-assembly-latest.jar report -l Kotlin -r ./guppy/build/reports/jacoco/jacocoTestReport/jacocoTestReport.xml"
```
Comment (AndrewHancock) [inline .travis.yml]: Is it safe to use the home directory expansion (~)? What about people who are not building in their home directory?

**Review by AndrewHancock — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (CCorrado) [reply .travis.yml]: this is just for travis, which is on a Linux server.
`2019-09-10 17:40:37 (38.6 MB/s) - ‘/home/travis/coverage/codacy-coverage-reporter-assembly-latest.jar’ saved [50491264/50491264]`

Check full output here: (Line 784) https://travis-ci.org/johnsonandjohnson/Guppy-Android/builds/583286341

File: .travis.yml [hunk 2]
```diff
@@ -14,9 +14,11 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
+    - $HOME/coverage/
 
 before_install:
   - yes | sdkmanager "platforms;android-28"
+  - wget -O ~/coverage/codacy-coverage-reporter-assembly-latest.jar https://github.com/codacy/codacy-coverage-reporter/releases/download/4.0.5/codacy-coverage-reporter-4.0.5-assembly.jar
```
Comment (Yakun1022) [inline .travis.yml]: should we be using a newer version of the codacy-coverage-reporter?

**Review by Yakun1022 — APPROVED** [review]
looks good!

**Pull request merged**